### DISCUSSION TITLE
Welcome to MDX_Vault Discussions!

### DISCUSSION THREAD

**Discussion by BobanSpasic** [BobanSpasic/MDX_Vault#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Piattica** [comment]
Fantastic idea!
My suggestions:
1)  it would be nice to have a zip for each bank
2) better to add date/release to each bank

**Reply by BobanSpasic** [reply]
Nice idea with the zips. I'll put it on my ToDo list.
